src/features/offer/pages/__snapshots__/Offer.test.tsx.snap

```diff
@@ -1081,8 +1077,176 @@ exports[`<Offer /> should match snapshot for digital offer 1`] = `
                             ]
                           }
                         >
-                          Où ?
-                        </Text>
+                          <Text
+                            style={
+                              Array [
+                                Object {
+                                  "color": "#151515",
+                                  "fontFamily": "Montserrat-Medium",
+                                  "fontSize": 18,
+                                  "lineHeight": 22,
+                                },
+                              ]
+                            }
+                          >
+                            Où ?
+                          </Text>
+                          <View>
+                            <Text
+                              style={
+                                Array [
+                                  Object {
+                                    "color": "#151515",
+                                    "fontFamily": "Montserrat-SemiBold",
+                                    "fontSize": 12,
+                                    "lineHeight": 16,
+                                    "paddingTop": 16,
+                                  },
+                                ]
+                              }
+                            >
+                              Adresse
+                            </Text>
+                            <Text
+                              style={
+                                Array [
+                                  Object {
+                                    "color": "#151515",
+                                    "fontFamily": "Montserrat-Regular",
+                                    "fontSize": 15,
+                                    "lineHeight": 20,
+                                    "paddingTop": 4,
+                                    "textTransform": "capitalize",
+                                  },
+                                ]
+                              }
+                            >
+                              Espace des Gnoux, RUE DES MORPHOS 97300 Cayenne
```

Comment:
On dirait qu'on a encore des parties de l'adresse en toutes majuscules dans les snapshots c'est assez étrange.

Reply:
en fait tu vois quand meme le textTransform capitalize au dessus, il met la data en brut dans le snap, style non appliqué mais noté

Reply:
Ahh ouiiii okay !